src/MissionParallelCatchup/parallel_catchup_helm/templates/catchup_workers.yaml

```diff
@@ -32,12 +32,32 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
+              {{- range .Values.worker.requireNodeLabels }}
+              - key: {{ .key }}
+                operator: {{ .operator }}
+                {{- with .values }}
+                values: {{ toJson . }}
+                {{- end }}
+              {{- end }}
+              {{- range .Values.worker.avoidNodeLabels }}
+              - key: {{ .key }}
+                operator: {{ .operator }}
+                {{- with .values }}
+                values: {{ toJson . }}
+                {{- end }}
+              {{- end }}
+              {{- if not .Values.worker.unevenSched }}
```

Comment:
This part works good but I noticed that the pre-existing selector interferes with karpenter we want to use for dynamic scaling.
Could you remove the following hardcoded selector:
```
              - key: node-role.kubernetes.io/control-plane
                operator: DoesNotExist
```
If we discover that it's needed at some point in the future we can add it dynamically.